includes/class-wc-payments.php

```diff
@@ -1223,4 +1225,16 @@ public static function init_platform_checkout() {
 			new Platform_Checkout_Tracker( self::get_wc_payments_http() );
 		}
 	}
+
+	/**
+	 * Load webpack runtime script, only if SCRIPT_DEBUG is enabled and the script exists.
+	 * Required for webpack server with HMR.
+	 *
+	 * @return void
+	 */
+	public static function enqueue_dev_runtime_scripts() {
+		if ( ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ) && file_exists( WCPAY_ABSPATH . 'dist/runtime.js' ) ) {
+			wp_enqueue_script( 'WCPAY_RUNTIME', plugins_url( 'dist/runtime.js', WCPAY_PLUGIN_FILE ), [], self::get_file_version( 'dist/runtime.js' ), true );
```

Comment:
I'm assuming `runtime.js` is created by HMR? My brief drive-by comment is to just check this new file can't get bundled up into the release ZIP accidentally. I think we destroy that `dist` folder before running the production build, so it's likely fine, but still worth double checking.

Reply:
That's right, [I've also added](https://github.com/Automattic/woocommerce-payments/blob/5d19129845e132624bb08cf8f7bb325803f04516/webpack/shared.js#L35) `output.clean` webpack config, to ensure `runtime.js` doesn't get loaded while not using HMR (EG `npm run start`).

Anyway, not sure what do you think of my previous comment. I'd like to look at it and see if we can get rid of this conditional script load in the PHP side. Although, I'll need some time before getting into it 🙂